examples/tutorials/t5_demo.py

```diff
@@ -374,3 +425,174 @@ def generate(encoder_tokens: Tensor, eos_idx: int, model: T5Model, beam_size: in
 #    review . School officials identified student during investigation and the person
 #    admitted to hanging the noose, Duke says . The noose, made of rope, was discovered on
 #    campus about 2 a.m.
+#
+
+
+#######################################################################
+# Generate Sentiment Classifications
+# ----------------------------------
+#
+# Similarly, we can now use the model to generate sentiment classifications on the first batch of reviews from the IMDB test set
+# using a beam size of 1.
+#
+
+batch = next(iter(imdb_dataloader))
+input_text = batch["text"]
+model_input = transform(input_text)
```

Comment:
nit: can we move this line to be just above `model_output`? Same with the other tasks